modules/nf-core/samtools/bam2fq/main.nf

```diff
@@ -39,17 +43,14 @@ process SAMTOOLS_BAM2FQ {
         """
         samtools \\
             bam2fq \\
+            $args \\
             -@ $task.cpus \\
-            -1 ${prefix}_1.bwa.fq.gz \\
-            -2 ${prefix}_2.bwa.fq.gz \\
-            -0 /dev/null \\
-            -s /dev/null \\
-            $bam
+            $inputbam | gzip --no-name > ${prefix}_interleaved.fq.gz
 
         cat <<-END_VERSIONS > versions.yml
```

Comment:
Can't you cat the versions after the if/else statement instead of repeating it twice here and above?

Reply:
if/else contains two script blocks. I guess cat can be done only inside script.
this is nf-core module btw